0007-managing-of-customer-apps/0007-managing-of-customer-apps.md

```diff
@@ -0,0 +1,7 @@
+# Managing of customer apps
+
+As a user I’d like to hand over the pager of my apps to Giant Swarm. 
```

Comment:
```suggestion
As a user I’d like to hand over the pager of my applications to Giant Swarm. 
```

I kind of use `applications` vs `apps` to differentiate customer's own applications vs upstream apps we manage. Need to update in glossary... https://intranet.giantswarm.io/docs/glossary/#app

Reply:
I try to use "workloads" here, to not get into any confusion and definition issues.